src/appkit.cs

```diff
@@ -23671,4 +23676,10 @@ interface NSOpenGLLayer
 		void Draw (NSOpenGLContext context, NSOpenGLPixelFormat pixelFormat, double t, ref CVTimeStamp ts);
 	}
 
+	[Protocol (IsInformal=true)]
```

Comment:
no, *only* `[Model]` should be using `[BaseType]` or the code generation is not correct (there's an bug for that)